bitnami/etcd/templates/snapshot-pvc.yaml

```
@@ -10,9 +10,13 @@ metadata:
   name: {{ printf "%s-snapshotter" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-" }}
   namespace: {{ .Release.Namespace | quote }}
   labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
-  {{- if .Values.commonAnnotations }}
-  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
-  {{- end }}
+  annotations:
+    {{- if .Values.commonAnnotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+    helm.sh/resource-policy: keep
+    {{- else }}
+    helm.sh/resource-policy: keep
+    {{- end }}
```

```suggestion
  annotations:
    helm.sh/resource-policy: keep
    {{- if .Values.commonAnnotations }}
    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
    {{- end }}
```

Also, you could parameterize it as in other charts to use something like (in that case we should use version 10.7.0 sin it would add new parameters).

```
  annotations:
    {{- if eq .Values.persistence.resourcePolicy "keep" }}
    helm.sh/resource-policy: keep
    {{- end }}
 ```

Since it is a minor change, it is up to you.